plyer/platforms/android/filechooser.py

```diff
@@ -141,12 +141,8 @@ def _on_activity_result(self, request_code, result_code, data):
         if request_code != self.select_code:
             return
 
-        # bad response
         if result_code != Activity.RESULT_OK:
```

Comment:
Is there a different result code to check for for the cancel case?

Reply:
Does not seem like it. `Activity.RESULT_OK` is always the result of a valid selection.

https://developer.android.com/training/basics/intents/result

Reply:
https://developer.android.com/training/basics/intents/result#ReceiveResult
It says you can use `RESULT_CANCELED` for that

Reply:
yes, but considering these are the only two alternatives, there is nothing to gain from testing that instead of `!= RESULT_OK` i did hope for the ability to distinguish user canceling from other errors, but either there are no other failure cases (which i doubt), or there is just no way to know.